desc/configuration.py

```diff
@@ -396,6 +396,12 @@ def __init__(
 
     # TODO: allow user to pass in arrays for surface, axis? or R_lmn etc?
     # TODO: make this kwargs instead?
+    def _set_up(self):
+        """Called after loading, to ensure object has all properties needed for current DESC version.
+        Allows for backwards-compatibility with equilibria saved/ran with older DESC versions"""
+        if not hasattr(self, "current"):
```

Comment:
A possibly more general fix would be something like:
```
for atttr in self._io_attrs_:
    if not hasattr(self, attr):
    self.attr = None
```
(i forget the correct syntax)

Reply:
good idea, changed it